include/nuttx/list.h

```diff
@@ -390,6 +390,11 @@ static inline bool list_is_empty(FAR struct list_node *list)
   return (list->next == list) ? true : false;
 }
 
+static inline bool list_is_clear(FAR struct list_node *list)
+{
+  return (list->next == NULL) ? true : false;
```

Comment:
```suggestion
  return (list->next == NULL);
```

Reply:
This just follow the style of list_is_empty, I think the consistent is more important.

Reply:
I think list_is_empty should be modified as well to keep the style. The result of `==` is already `true` or `false` so usage of the trinary operator here is redundant.

Reply:
Done.